### ISSUE TITLE
Discarding returns the same MailerResponse.android as sending

### ISSUE THREAD

**Issue by deakjahn** [taljacobson/flutter_mailer#33]
On Android (10) there is no difference whether the mail was sent or discarded, the response is the same.

**Comment by taljacobson** [comment]
Yes, that is true, and unfortunately the only option on android, only ios mail has the ability to return if the mail was discarded send or cancelled.

**Comment by deakjahn** [comment]
Then I would suggest to add a word about it to the comments in the README and then people like me won't bother you with it any more. :-))

**Comment by taljacobson** [comment]
I believe both the readme example and comments on the code mention `ios only`, but there might be a better way to mitigate further confusion.

**Comment by deakjahn** [comment]
In hindsight, maybe. I'd just add to the `.android` case that both sent and discarded come back that way.
